src/reference/asciidoc/amqp.adoc

```diff
@@ -472,6 +472,9 @@ Starting with Spring AMQP _version 1.5_, these properties can be set directly on
 If both discrete properties and `sslPropertiesLocation` is provided, properties in the latter will override the
 discrete values.
 
+IMPORTANT: Starting with version 2.0, the server certificate is validated by default because it is more secure. If you wish to skip this validation for some reason, set the factory bean's `skipServerCertificateValidation` property to `true`.
+Starting with version 2.1, the `RabbitConnectionFactoryBean` now calls `enableHostnameVerification()` by default; to revert to the previous behavior, set the `enableHostnameVerification` property to `false`.
```

Comment:
I guess we need just remove this sentence in back-port commit: there is just no version `2.1` for `2.0` to mention in its docs.

Reply:
Ah! I understand now you in your commit message.

Sorry: merging with mentioned polishing and proper back-port changes....

Reply:
See my comment in the PR

>but change the second sentence to

>Also, the RabbitConnectionFactoryBean will call enableHostnameVerification() if you set the enableHostnameVerification property to true; this will be the default in version 2.1.

Reply:
(But it's the third sentence 😄  )

Reply:
He-he! Indeed 😉